pkgs/misc/themes/vertex/default.nix

```diff
@@ -1,33 +1,38 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, gtk3, pkgconfig }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
 
 stdenv.mkDerivation rec {
-  name = "theme-vertex-${version}";
-  version = "20160329";
+  name = "${pname}-${version}";
```

Comment:
@rycee I think that following the upstream name (`vertex-theme`) is better. But it is not that important and I can revert to the previous name. I will make a PR for that.